client/util/routes.ts

```diff
@@ -37,6 +37,10 @@ const postObjectDatastreamUrl = (pid) => {
     return `${editObjectUrl}/${pid}`;
 }
 
+const deleteObjectDatastreamUrl = (pid, datastream: string) => {
+    return `${editObjectUrl}/${pid}/datastream/${datastream.toLowerCase()}`;
```

Comment:
Why are you lower-casing the datastream name here and then uppercasing it again in the route? While in practice all datastream names are currently ALLCAPS, I think it might be safer to assume that they are case-sensitive for future flexibility.

Reply:
I think it was a stylistic choice since I don't usually see CAPS LOCK characters in routes. But, after investigating how CAPS LOCK characters in urls would affect the app, I'm just going to remove the lowercase and uppercase operations.